stdlib/universe/meta_query_keys_test.flux

```diff
@@ -81,6 +81,7 @@ t_meta_query_keys = (table=<-) => {
             |> range(start: 2018-05-22T19:53:26Z)
             |> filter(fn: (r) => r._measurement == "cpu")
             |> keys()
+            |> sort()
```

Comment:
The old map ordered the columns by using the sorted properties. This isn't necessarily correct. It's really supposed to be the order that the type system tells us. The new map used what the type system tells us about the order. These tests produce values depending on the column order which, in my opinion, should be treated as non-deterministic since the only reason why the order differed was because a map was inserted.